### ISSUE TITLE
Allow to configure default gitignore's + header separation

### ISSUE THREAD

**Issue by skwde** [CodeZombieCH/vscode-gitignore#48]
It would be nice 

- if one could configure default gitignore's to add when creating one.
- have a some kind of seperator comment string to distinguish between the different section.

Specifying something like

```jsonc
gitignore.defaults = ['Global/Windows.gitignore', 'Global/Linux.gitignore', 'Global/MacOS.gitignore']
```

in `settings.json`  should add those to the gitignore by default.

The header for the MacOS section could look something like

```
###############################################
# ## Global/MacOS.gitignore
```

**Comment by pythoninthegrass** [comment]
Agreed. I primarily develop on mac, so that's the base `.gitignore`. Think it'd be useful to add custom `.gitignore` files from either a public repo/gist or a local file.
